models/Challenge.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const queries = require('./queries');
+const utils = require('../utils');
+
+class Challenge {
+  makeChallenge(userId, spotId) {
+    const sql = queries.makeChallenge;
+
+    return new Promise((resolve, reject) => {
+      const client = utils.db.createClient();
+
+      client.connect();
```

Comment:
A connection to the db is opened for every function call to our model classes. At a larger scale, that will probably lead to some performance issues. If it actually leads to a problem, I'll optimize